src/analysis/normed_space/pi_Lp.lean

```diff
@@ -324,34 +534,39 @@ subtype.ext $ by { push_cast, exact dist_eq_of_L2 _ _ }
 
 lemma edist_eq_of_L2 {β : ι → Type*} [Π i, seminormed_add_comm_group (β i)] (x y : pi_Lp 2 β) :
   edist x y = (∑ i, edist (x i) (y i) ^ 2) ^ (1 / 2 : ℝ) :=
-by simp_rw [pi_Lp.edist_eq, ennreal.rpow_two]
-
-include fact_one_le_p
+by simp [pi_Lp.edist_eq_sum]
 
 variables [normed_field 𝕜]
 
+-- this was necessary to get Lean to accept `∥c • f∥₊` in the `normed_space` instance below
+-- can we just do this with `letI`?
```

Comment:
Yes, you can remove this instance if you put `letI: module 𝕜 (pi_Lp ⊤ β) := pi.module ι β 𝕜,` on Line 552, and if on Line 569 you switch back from `module` to `normed_space` (which makes sense: you will never take the L^p product of bare modules, you will always have a norm).

Reply:
@sgouezel:

> which makes sense: you will never take the L^p product of bare modules, you will always have a norm.

Or a quasi-norm if `p < 1`, which would be a different structure, and in that case it would make sense to have the module declaration separate (so that it works for both `p < 1` and `1 ≤ p`). However, since the `add_comm_group` instance is currently folded into the `seminormed_add_comm_group` instance, I'll fold the module one into normed space too. These can be separated when someone comes along to do the quasi-norm structures.